OPCBUGS-36928#Enabling swap node clarification (#78987)
openshift/openshift-docs: modules/nodes-nodes-swap-memory.adoc

:FeatureName: Enabling swap memory use on nodes
include::snippets/technology-preview.adoc[]

> Except for container-native virtualization (CNV) users or use cases, enabling swap memory is not available to {product-title} users.

Reviewer: ```suggestion
Enabling swap memory is only available for container-native virtualization (CNV) users or use cases.
```

Reviewer: Hello @brendan-daly-red-hat,
   Reading through the original JIRA ticket, I think this suggestion would make sense. Also, I wonder if the placement should be in some sort of call out?  I realize the updated content is currently in between two call outs already, but at first glance it seems like it is an important enough detail to be made more prominent.
